### DISCUSSION TITLE
user experience feedback

### DISCUSSION THREAD

**Discussion by Slaneche** [liftoff-sr/CIPster#32] [Q&A] [answered]
Hello, 

is there any feedback on this stack ?

I tried OpENer with EtherNet/IP Explorer, with success (under Windows)
but with this stack, (with many stack code changes for make it work), I have a small success (on ExclusiveOwner work, but does not work on InputOnly).

Did I miss something ? or this stack has a lot of issue?

**Comment by liftoff-sr** [comment, marked as answer]
You may be confusing the application layer example with the stack.  The stack is more than the sample applications.  My company uses the stack extensively in many installations without issue.  So no, this stack does not have a lot of issues and has 86 github stars.  You can assume that many of those stars are by folks actually using the stack, but of course they have written their own applications and then linked to the stack library.
